support/gathering-cluster-data.adoc

```diff
@@ -21,3 +21,22 @@ include::modules/support-gather-data.adoc[leveloffset=+1]
 
 // Obtain your cluster identifier
 include::modules/support-get-cluster-id.adoc[leveloffset=+1]
+
+// About sosreport
+include::modules/about-sosreport.adoc[leveloffset=+1]
+
+// Generating a sosreport
+include::modules/support-generating-a-sosreport-archive.adoc[leveloffset=+1]
+
+// Querying bootstrap node journal logs
+include::modules/querying-bootstrap-node-journal-logs.adoc[leveloffset=+1]
+
+// Querying cluster node journal logs
+include::modules/querying-cluster-node-journal-logs.adoc[leveloffset=+1]
+
+// Collecting a network trace from an OpenShift cluster node
+include::modules/support-collecting-network-trace.adoc[leveloffset=+1]
+
+// TODO: Extracting debug data from a cluster
```

Comment:
Where does must-gather fit into this?

Reply:
I have now removed this comment from within `modules/upi-installation-considerations.adoc`, because we have covered this topic extensively, including the `must-gather` sections in the Support book.